components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -73,6 +73,7 @@ class ToolbarPresenter(
         updateToolbarSecurity(session?.securityInfo ?: Session.SecurityInfo())
 
         if (session != null) {
+            toolbar.isExternal = session.isCustomTabSession()
```

Comment:
I'd leave this to the app code. Fenix already configures its toolbar for normal / custom tab browsing. In that code I'd add the code to show the divider. Does this make sense?

Reply:
Updated ✅